pipeline/jobs/fill_missing_months.py

```diff
@@ -0,0 +1,71 @@
+from pyspark.sql import DataFrame, functions as F
+
+
+def fill_blanks_with_zeros(
+    df: DataFrame,
+    date_column: str = "SpendMonth",
+    amount_column: str = "EvidencedSpend",
+    columns_to_consider: list[str] = [],
+) -> DataFrame:
+    """Receive a dataframe and return a new dataframe with zero-amount placeholder rows added for every missing months"""
```

Comment:
Update all method comments to the below format:

"""Receive a dataframe and return a new dataframe with zero-amount placeholder rows added for every missing months
    Args:
        df: Input DataFrame:
        date_column: ...
    Returns:
        Dataframe with no missing month
    """